components/collection/drop/MintStepper.vue

```diff
@@ -4,6 +4,7 @@
     v-model="amountToMint"
     data-testid="drop-stepper-container"
     :max="max"
+    :min="1"
```

Comment:
What happens if it's the`HolderOf` drop? The amount may be zero.

Reply:
update it